spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  def response_body
+    JSON.parse(response.body)
+  end
+
+  def create_cuisine
+    create(:cuisine, name: 'C1')
+  end
+
+  let(:restaurant1) {create(:restaurant, name: 'R1')}
+  let(:restaurant2) {create(:restaurant, name: 'R2')}
+  let(:restaurant3) {create(:restaurant, name: 'R3')}
+  let(:restaurants) { [restaurant1, restaurant2, restaurant3]}
+  let(:rest_params) {FactoryBot.attributes_for(:restaurant)}
+  let(:rest_no_name) {{cuisine_id: 1, delivery_time: 50}}
+  let(:rest_no_cuisine) {{name: 'Rest', delivery_time: 50}}
+
+
+
+  describe 'Index' do
```

Comment:
the convention it to not go with the capital letter on specs descriptions. I know it sounds weird but that's the common pattern.